scipy/misc/_common.py

```diff
@@ -226,7 +226,9 @@ def face(gray=False):
     face = frombuffer(data, dtype='uint8')
     face.shape = (768, 1024, 3)
     if gray is True:
-        face = (0.21 * face[:,:,0] + 0.71 * face[:,:,1] + 0.07 * face[:,:,2]).astype('uint8')
+        face = (0.21 * face[:,:,0] + 
+                0.71 * face[:,:,1] + 
+                0.07 * face[:,:,2]).astype('uint8')
```

Comment:
PEP8 actually recommends
```suggestion
        face = (0.21 * face[:, :, 0]
                + 0.71 * face[:, :, 1]
                + 0.07 * face[:, :, 2]).astype('uint8')
```

https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator